# Call
std_msgs/String task
std_msgs/String req
---
# Return
zeabus_vision/vision_drum data
